backtrace-library/src/main/java/backtraceio/library/services/BacktraceMetrics.java

```diff
@@ -360,8 +378,8 @@ private boolean shouldProcessEvent(String name) {
         return true;
     }
 
-    protected Map<String, Object> createLocalAttributes(Map<String, Object> attributes) {
-        Map<String, Object> localAttributes = new HashMap<String, Object>();
+    Map<String, Object> createLocalAttributes(Map<String, Object> attributes) {
```

Comment:
can we keep it obvious? I think its fine to leave protected modified here. I know its default behavior but we keep accessor definition everywhere.

Reply:
fixed